Proy/test.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from game import draw, update_board, get_valid_input, winner, play_user, play_computer, start
+
+
+def test_update_board():
+    game = [[" ", " ", " "], [" ", " ", " "], [" ", " ", " "]]
```

Comment:
-> easier to understand
```suggestion
    game = [["x", "o", "x"], 
            ["o", "x", "o"], 
            ["x", "o", "x"]]
```